src/services/rfqm_service.ts

```diff
@@ -0,0 +1,95 @@
+import { MarketOperation, QuoteRequestor } from '@0x/asset-swapper';
+import { BigNumber } from '@0x/utils';
+
+import { RFQT_REQUEST_MAX_RESPONSE_MS } from '../config';
+import { NULL_ADDRESS } from '../constants';
+
+export interface FetchIndicativeQuoteParams {
```

Comment:
Could we move these to `types.ts`?

Reply:
So, I had these in `types.ts` but after addressing a previous comment, moved them here: https://github.com/0xProject/0x-api/pull/535#discussion_r615051010

Happy to do either one, but would be ideal to settle on a single convention